### DISCUSSION TITLE
Adetailer section not showing after install and restart

### DISCUSSION THREAD

**Discussion by magickaito** [Bing-su/adetailer#268] [Q&A]
![Screenshot 2023-08-11 at 12 33 00 AM](https://github.com/Bing-su/adetailer/assets/1195573/b7da407d-951c-43c1-bd5c-c195f2f09a1b)

Hi guys, I am using Stable Diffusion on Runpod, I have installe this adetailer, and restarted the pod entirely. 
But I couldn't find any adetailer section:

![Screenshot 2023-08-11 at 12 32 39 AM](https://github.com/Bing-su/adetailer/assets/1195573/b0fabae2-20b3-4d3f-84cf-a035b158c4f1)

Is there anything else I need to do to turn on the section?

**Comment by magickaito** [comment]
I found this error log:

2023-08-10T16:23:54.716501583Z     ModuleNotFoundError: No module named 'ultralytics'`

Looks like I need a custom runpod template to use this....

**Comment by Bing-su** [comment]
runpod에서는 항상 이 문제가 발생합니다... 
https://github.com/Bing-su/adetailer/discussions/205
